.travis.yml

```diff
@@ -45,17 +36,55 @@ before_script:
 script:
   - make test-coverage codecov
 
-before_deploy:
-  - make packages
+jobs:
+  include:
+    - go: 1.10.x
+      os: linux
+      sudo: required
+      dist: trusty
+      services: [docker]
+
+      before_deploy:
+        - make packages
+
+      after_deploy:
+        - DOCKER_PUSH_LATEST=1 make docker-push
+
+      deploy:
+        provider: releases
+        api_key: $GITHUB_TOKEN
+        file_glob: true
+        file: build/*linux_amd64.tar.gz
+        skip_cleanup: true
+        on:
+          tags: true
+
+    - {go: 1.9.x, os: linux, sudo: required, dist: trusty, services: [docker]}
+    
+    - {go: tip, os: linux, sudo: required, dist: trusty, services: [docker]}
+
+    - go: 1.10.x
+      os: osx
+      osx_image: xcode9.3
+
+      before_install:
+        - echo "skipping before_install for macOS"
+
+      before_script:
+        - echo "skipping before_script for macOS"
 
-deploy:
-  provider: releases
-  api_key: $GITHUB_TOKEN
-  file_glob: true
-  file: build/*.tar.gz
-  skip_cleanup: true
-  on:
-    tags: true
+      script:
+        - make packages || echo "" # will fail because of docker being missing
```

Comment:
I think make packages will print an error if Docker is missing, but will not fail.

Reply:
@erizocosmico ^^^

Reply:
Yes, it does fail, I had to add `|| echo ""` because it did make it fail :( (it would be `|| true`, but macOS travis does not have `true`)